src/lp/events.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * LP specific events.
+ */
+export enum LiquidityEventName {
+  ADD_LIQUIDITY = 'Add Liquidity',
+  COLLECT_LIQUIDITY = 'Collect Liquidity',
+  MIGRATE_LIQUIDITY = 'Migrate Liquidity',
+  REMOVE_LIQUIDITY = 'Remove Liquidity',
```

Comment:
Are these submitting to chain or confirmed?

Reply:
these were being sent when submitted to the chain